exporting/read_config.c

```diff
@@ -238,8 +238,11 @@ struct engine *read_exporting_config()
         prometheus_exporter_instance->config.update_every =
             prometheus_config_get_number(EXPORTING_UPDATE_EVERY_OPTION_NAME, EXPORTING_UPDATE_EVERY_DEFAULT);
 
-        while(!global_backend_source)
-            usleep(100);
+        for (int retries = 0; !global_backend_source && retries < 1000; retries++)
+            usleep(10000);
```

Comment:
Please, instead to use `usleep` use `sleep_usec(usec_t usec)`.

Reply:
Fixed.